openwisp_notifications/checks.py

```diff
@@ -27,3 +27,32 @@ def check_cors_configuration(app_configs, **kwargs):
             )
         )
     return errors
+
+
+@checks.register
+def check_ow_object_notification_widget_setting(app_configs, **kwargs):
+    errors = []
+    if not isinstance(app_settings.OW_OBJECT_NOTIFICATION_WIDGET, list):
+        errors.append(
+            checks.Warning(
+                msg='Improperly Configured',
+                hint=(
+                    'OW_OBJECT_NOTIFICATION_WIDGET should be a list,'
+                    f' {type(app_settings.OW_OBJECT_NOTIFICATION_WIDGET)} provided'
+                ),
+                obj='Settings',
+            )
+        )
+    for model_admin in app_settings.OW_OBJECT_NOTIFICATION_WIDGET:
```

Comment:
model_admin is technically not an instance of `ModelAdmin` but a python `path`, while reading this code I thought it was checking if the path was an instance of `ModelAdmin` and got confused.

I think it would make the code more readable to rename `model_admin` to `path`.